sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -83,7 +83,7 @@ Set-SPOTenant [-ApplyAppEnforcedRestrictionsToAdHocRecipients <Boolean>]
  [-SharingDomainRestrictionMode <SharingDomainRestrictionModes>]
  [-ShowPeoplePickerSuggestionsForGuestUsers <Boolean>]
  [-SpecialCharactersStateInFileFolderNames <SpecialCharactersState>]
- [-SyncPrivacyProfileProperties <Boolean>]
+ [-ReSyncTenantPrivacyProfile <Boolean>]
```

Comment:
Not boolean I think:

![image](https://user-images.githubusercontent.com/33589238/182382238-5f219ec0-0001-4f57-92f7-81089ea0cc17.png)

Reply:
Just tested it as well. It's not a Boolean and don't need any additional parameters.

Reply:
Updated it but I'm usure on the Type too.